wayshot/src/utils.rs

```diff
@@ -139,5 +139,35 @@ pub fn get_default_file_name(extension: EncodingFormat) -> PathBuf {
         .map(|time| time.as_secs().to_string())
         .unwrap_or("unknown".into());
 
-    format!("{time}-wayshot.{extension}").into()
+    format!("wayshot-{time}.{extension}").into()
+}
+
+fn get_hour_minute_from_unix_seconds(seconds: u64) -> String {
+    let total_minutes = seconds / 60;
+    let mut current_hour = (((total_minutes / 60) % 24) + 5) % 24;
+    let mut current_minute = (total_minutes % 60) + 30;
+
+    if current_minute > 60 {
+        current_hour += 1;
+    }
+
+    current_minute = current_minute % 60;
+
+    if current_hour == 24 {
+        current_hour = 0;
+    }
+
+    format!("{}:{}:{}", current_hour, current_minute, seconds % 60)
+}
```

Comment:
This is only printing time `HH:MM:SS` when we need the date and time. Though this actually made me think this format might be quite annoying to use because of the colons and possibly spaces: `DD-MM-YY HH:MM:SS`. Could we go with `2024-12-31 23:59:59`? @Shinyzenith

Additionally, this function can be removed and we could use the `chrono` package. Considering this is a binary, adding a dependency for time stuff is fine.

See the docs here: https://docs.rs/chrono/latest/chrono/format/index.html

Reply:
> This is only printing time HH:MM:SS when we need the date and time. Though this actually made me think this format might be quite annoying to use because of the colons and possibly spaces: DD-MM-YY HH:MM:SS. Could we go with 2024-12-31 23:59:59? @Shinyzenith

That is fine by me, doesn't seem to invasive and is pretty easy to read / reason through.

> Additionally, this function can be removed and we could use the chrono package. Considering this is a binary, adding a dependency for time stuff is fine.

I initially had suggested not to increase the dependency graph but if you feel so, we can definitely go forth with `chrono`. Apologies for the extra work 😅 @rachancheet .